powershell/Tests/L0/_suite.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference path="../../definitions/mocha.d.ts"/>
+/// <reference path="../../definitions/node.d.ts"/>
+/// <reference path="../../definitions/Q.d.ts"/>
+
+import Q = require('q');
+import assert = require('assert');
+import psm = require('../lib/psRunner');
+import path = require('path');
+import fs = require('fs');
+var shell = require('shelljs');
+var ps = shell.which('powershell');
+describe('VstsTaskSdk Suite', function () {
+    this.timeout(20000);
+
+    before((done) => {
+        // init here
+        done();
+    });
+
+    after(function () {
+    });
+
+    if (ps) {
+        fs.readdirSync(__dirname).forEach(function (file: string) {
```

Comment:
Here I'm just doing foreach ps1 file in the directory, run the script via psm.runPS( [...] ). This is just a convenience so we don't miss any files.